dockers/docker-database/supervisord.conf.j2

```diff
@@ -20,7 +20,7 @@ stderr_logfile=syslog
 {% if INSTANCES %}
 {%     for redis_inst, redis_items in INSTANCES.iteritems() %}
 [program: {{ redis_inst }}]
-command=/bin/bash -c "{ [[ -s /var/lib/{{ redis_inst }}/dump.rdb ]] || rm -f /var/lib/{{ redis_inst }}/dump.rdb; } && mkdir -p /var/lib/{{ redis_inst }} && exec /usr/bin/redis-server /etc/redis/redis.conf --port {{ redis_items['port'] }} --unixsocket {{ redis_items['unix_socket_path'] }} --pidfile /var/run/redis/{{ redis_inst }}.pid --dir /var/lib/{{ redis_inst }}"
+command=/bin/bash -c "{ [[ -s /var/lib/{{ redis_inst }}/dump.rdb ]] || rm -f /var/lib/{{ redis_inst }}/dump.rdb; } && mkdir -p /var/lib/{{ redis_inst }} && exec /usr/bin/redis-server /etc/redis/redis.conf --bind {{ LOOPBACK_IP }} {{ redis_items['hostname'] }} --port {{ redis_items['port'] }} --unixsocket {{ redis_items['unix_socket_path'] }} --pidfile /var/run/redis/{{ redis_inst }}.pid --dir /var/lib/{{ redis_inst }}"
```

Comment:
>LOOPBACK_IP [](start = 215, length = 11)

Did you expose the Redis endpoint to public?

Reply:
With this change I am trying to do two things 
  (i) redis server on the linux host will be listening only on 127.0.0.1 address ( as it exists now )
  (ii) redis server in the namespaces "asic0", "asic1" etc will be listening on 127.0.0.1, 240.1.1.* 
       (address which is on eth0 and connected to docker0 bridge in the host )

As per my understanding from a bit of research I see that the docker by default adds DROP rules for IP address inside containers (https://github.com/moby/moby/issues/34164). 

But looks like in SONIC when we start the docker daemon we set the iptables = false, which would not result in docker daemon not making the DROP rules ...in SONIC. 

Hence I think I agree with you that this port could be exposed to public if some one uses this IP subnet explicitly. One good part though is that this IP address range 240.* which we use is a class E address which ideally should not be used in pubic. So we are safe unless a hacker tries to use this IP address range explicitly.

Let me know your thoughts.

Reply:
The subnet 240.127.1.1 IP is already configured for docker0 in the single ASIC platforms deployed.  This could be targeted by outside traffic. 
So it appears to me that this is a day one issue which we could handle it by adding an iptables rule to block external traffic coming into the control plane for this dest subnet. 

I am planning to do some more testing to see if the traffic destined to this subnet really reaches the linux host from management port/front-panel port. But would prefer to take this up as a separate enhancement, if it is ok ?